commit 514de1ec0d390c1069298c61a31d878a7b8e72e2 (refs/changes/78/3378/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-02-15T17:24:28-08:00 (1 year, 8 months ago)
    
    TRITON-156 clean some cruft out of sdc-config-agent.git

diff --git a/.gitmodules b/.gitmodules
index 04abb3e..1d72cc4 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -7,6 +7,3 @@
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
 	url = https://github.com/davepacheco/jsstyle.git
-[submodule "deps/sdc-scripts"]
-	path = deps/sdc-scripts
-	url = https://github.com/joyent/sdc-scripts.git
diff --git a/Makefile b/Makefile
index 841fa00..444d76a 100644
--- a/Makefile
+++ b/Makefile
@@ -44,7 +44,7 @@ include ./tools/mk/Makefile.smf.defs
 # Repo-specific targets
 #
 .PHONY: all
-all: $(SMF_MANIFESTS) | $(NPM_EXEC) $(REPO_DEPS) sdc-scripts
+all: $(SMF_MANIFESTS) | $(NPM_EXEC)
 	$(NPM) install && ./node_modules/.bin/kthxbai
 
 DISTCLEAN_FILES+=node_modules
@@ -116,5 +116,3 @@ ifeq ($(shell uname -s),SunOS)
 endif
 include ./tools/mk/Makefile.smf.targ
 include ./tools/mk/Makefile.targ
-
-sdc-scripts: deps/sdc-scripts/.git
diff --git a/deps/sdc-scripts b/deps/sdc-scripts
deleted file mode 160000
index 1250104..0000000
--- a/deps/sdc-scripts
+++ /dev/null
@@ -1 +0,0 @@
-Subproject commit 1250104a1defac4ea6f2cbbbe9b4b8313a543e1b
diff --git a/etc/agentconfig.json b/etc/agentconfig.json
deleted file mode 100644
index 60dec77..0000000
--- a/etc/agentconfig.json
+++ /dev/null
@@ -1,12 +0,0 @@
-{
-	"logLevel": "debug",
-	"pollInterval": 5000,
-	"sapi": {
-		"url": "http://10.2.206.24"
-	},
-	"localManifestDirs": [
-		"/opt/smartdc/sapi",
-		"/opt/smartdc/sapi/sapi_manifests",
-		"/var/tmp/does_not_exist"
-	]
-}
diff --git a/tools/mk/Makefile.node_deps.defs b/tools/mk/Makefile.node_deps.defs
deleted file mode 100644
index 29a83f7..0000000
--- a/tools/mk/Makefile.node_deps.defs
+++ /dev/null
@@ -1,43 +0,0 @@
-# -*- mode: makefile -*-
-#
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-#
-
-#
-# Copyright (c) 2014, Joyent, Inc.
-#
-
-#
-# Makefile.node_deps.defs: Makefile for including npm modules whose sources
-# reside inside the repo.  This should NOT be used for modules in the npm
-# public repo or modules that could be specified with git SHAs.
-#
-# NOTE: This makefile comes from the "eng" repo. It's designed to be dropped
-# into other repos as-is without requiring any modifications. If you find
-# yourself changing this file, you should instead update the original copy in
-# eng.git and then update your repo to use the new version.
-#
-
-#
-# This Makefile takes as input the following make variable:
-#
-#    REPO_MODULES	List of relative paths to node modules (i.e., npm
-#    			packages) inside this repo.  For example:
-#    			src/node-canative, where there's a binary npm package
-#    			in src/node-canative.
-#
-# Based on the above, this Makefile defines the following new variables:
-#
-#    REPO_DEPS		List of relative paths to the installed modules.  For
-#    			example: "node_modules/canative".
-#
-# The accompanying Makefile.node_deps.targ defines a target that will install
-# each of REPO_MODULES into REPO_DEPS and remove REPO_DEPS with "make clean".
-# The top-level Makefile is responsible for depending on REPO_DEPS where
-# appropriate (usually the "deps" or "all" target).
-#
-
-REPO_DEPS    = $(REPO_MODULES:src/node-%=node_modules/%)
-CLEAN_FILES += $(REPO_DEPS)
diff --git a/tools/mk/Makefile.node_deps.targ b/tools/mk/Makefile.node_deps.targ
deleted file mode 100644
index bb2ab4f..0000000
--- a/tools/mk/Makefile.node_deps.targ
+++ /dev/null
@@ -1,24 +0,0 @@
-# -*- mode: makefile -*-
-#
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at http://mozilla.org/MPL/2.0/.
-#
-
-#
-# Copyright (c) 2014, Joyent, Inc.
-#
-
-#
-# Makefile.node_deps.targ: targets for Makefile.node_deps.defs.
-#
-# NOTE: This makefile comes from the "eng" repo. It's designed to be dropped
-# into other repos as-is without requiring any modifications. If you find
-# yourself changing this file, you should instead update the original copy in
-# eng.git and then update your repo to use the new version.
-#
-
-NPM_EXEC  ?= $(error NPM_EXEC must be defined for Makefile.node_deps.targ)
-
-node_modules/%: src/node-% | $(NPM_EXEC)
-	$(NPM) install $<
